Diary/Extension/UIViewController+Share.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UIViewController+Share.swift
+//  Diary
+//
+//  Created by Seoyeon Hong on 2023/05/05.
+//
+
+import UIKit
+
+extension UIViewController {
+    func shareText(_ textToShare: String?) {
```

Comment:
```suggestion
    func share(someText: String) {
```

아 그리구 suggestion은 정말 제안이지 고치라는 의미는 아닙니다~
제 취향 강요임 ㅎㅎ..